client/src/components/Nav/NewChat.tsx

```diff
@@ -90,33 +90,29 @@ export default function NewChat({
   const navigate = useNavigate();
   const localize = useLocalize();
   const { conversation } = store.useCreateConversationAtom(index);
+  const defaultPreset = useRecoilValue(store.defaultPreset);
 
-  const clickHandler = useCallback(
-    (event: React.MouseEvent<HTMLAnchorElement>) => {
-      if (event.button === 0 && !(event.ctrlKey || event.metaKey)) {
-        event.preventDefault();
-        queryClient.setQueryData<TMessage[]>(
-          [QueryKeys.messages, conversation?.conversationId ?? Constants.NEW_CONVO],
-          [],
-        );
-        newConvo();
-        navigate('/c/new');
-        toggleNav();
-      }
-    },
-    [queryClient, conversation, newConvo, navigate, toggleNav],
-  );
+  const clickHandler = useCallback(() => {
+    queryClient.setQueryData<TMessage[]>(
+      [QueryKeys.messages, conversation?.conversationId ?? Constants.NEW_CONVO],
+      [],
+    );
+    const params = createChatSearchParams(defaultPreset ?? conversation);
+    const newRoute = params.size > 0 ? `/c/new?${params.toString()}` : '/c/new';
+
+    newConvo();
+    navigate(newRoute);
+    toggleNav();
+  }, [queryClient, conversation, newConvo, navigate, toggleNav, defaultPreset]);
 
   return (
     <div className="sticky left-0 right-0 top-0 z-50 bg-surface-primary-alt pt-3.5">
       <div className="pb-0.5 last:pb-0" style={{ transform: 'none' }}>
-        <a
-          href="/"
-          tabIndex={0}
+        <button
           data-testid="nav-new-chat-button"
```

Comment:
Moved this to a button since it's not linking off the page. I didn't see any issues with it when I tested it. Semantically a button is correct here.